searx/engines/brave.py

```diff
@@ -397,19 +396,26 @@ def fetch_traits(engine_traits: EngineTraits):
 
     # search regions of brave
 
-    engine_traits.all_locale = 'all'
+    resp = get('https://cdn.search.brave.com/serp/v2/_app/immutable/chunks/parameters.734c106a.js', headers=headers)
```

Comment:
This could probably break really quick again, but I don't have a better idea unfortunately

Reply:
Yes, I have checked: the URL is not a href in a `<link>` of the root html from https://search.brave.com/settings .. this doc loads some JS and one of the loaded JS scripts extends the `<link>` list in the root document to load the `parameters.734c106a.js'` .. very complicated.  Instead of having two hard to track indirectness I decided to read directly from `parameters.734c106a.js` .. and see when it breaks :o